app/src/forms/admin/routes.js

```diff
@@ -42,21 +195,193 @@ routes.get('/forms/:formId/versions/:formVersionId', async (req, res, next) => {
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ *      '404':
+ *        $ref: '#/components/responses/Error/ResourceNotFound'
+ * */
 routes.get('/forms/:formId/formUsers', async (req, res, next) => {
   await controller.getFormUserRoles(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/forms/{formId}/addUser:
+ *  put:
+ *   tags:
+ *    - Admin
+ *   summary: Add user permissions to a form
+ *   description: This endpoint will add users and roles/permissions to the form.
+ *   security:
+ *    - openId: []
+ *   parameters:
+ *    - in: path
+ *      name: formId
+ *      schema:
+ *       type: string
+ *       format: uuid
+ *      description: ID of the form.
+ *      required: true
+ *      example: c6455376-382c-439d-a811-0381a012d696
+ *    - in: query
+ *      name: formId
+ *      schema:
+ *       type: string
+ *       format: uuid
+ *      description: ID of the form.
+ *      required: true
+ *      example: c6455376-382c-439d-a811-0381a012d696
+ *    - in: query
+ *      name: userId
+ *      schema:
+ *       type: string
+ *       format: uuid
+ *      description: ID of the User.
+ *      required: true
+ *      example: 7cc11a0f-a7c9-4cbe-bd92-a2a22781ad9f
+ *   requestBody:
+ *    required: true
+ *    content:
+ *      application/json:
+ *        schema:
+ *          type: object
+ *          properties:
+ *            formId:
+ *              type: string
+ *              description: ID of the form
+ *              format: uuid
+ *              example: c6455376-382c-439d-a811-0381a012d696
+ *            userId:
+ *              type: string
+ *              format: uuid
+ *              description: ID of the User. This Id is from the User table in database.
+ *              required: true
+ *              example: 7cc11a0f-a7c9-4cbe-bd92-a2a22781ad9f
+ *            role:
+ *              type: string
+ *              description: Value must be 'owner'
+ *              default: owner
+ *              required: true
+ *              example: owner
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminSetFormUserRoles'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '404':
+ *        $ref: '#/components/responses/Error/ResourceNotFound'
+ *    '422':
+ *      description: 'Unprocessable Entity'
+ *      content:
+ *        application/json:
+ *          schema:
+ *            oneOf:
+ *              - $ref: '#/components/schemas/respError/UnprocessableEntityDBError'
+ *              - $ref: '#/components/schemas/respError/RoleRequiredValidationError'
+ *              - $ref: '#/components/schemas/respError/UserAndFormIdequiredError'
```

Comment:
In the UI it looks like this:

![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/d8b7e5f8-ce82-4e47-aefc-fe5365a5f2f5)

but the three "Object" buttons all display the same "Problem" default values. And in the right hand side Response Samples, only the last one (UserAndFormIdequiredError) is shown.

Is there any value in sorting this out? The developer might want to know that there are other errors possible, and not just UserAndFormIdequiredError.

Reply:
![Screenshot 2023-07-05 161912](https://github.com/bcgov/common-hosted-form-service/assets/14250649/9260cd9a-9791-4c54-8309-e2d90edb8dea)
This is how it is displaying on my end. I will switch to swagger ui to see why it is not showing other errors